### PR TITLE
Add a test for synthesized get-only properties in swiftinterfaces

### PR DESCRIPTION
**Pull request by jrose-apple**
Probably superfluous with all the other tests @harlanhaskins has been adding, but I wrote it and it was sitting around on my machine, so we might as well use it.

### CODE REVIEW

**Comment by jrose-apple** [comment]
@swift-ci Please test

**Comment by harlanhaskins** [comment]
Thanks! I don't think these are tested in the other tests.

**Review by harlanhaskins — APPROVED** [review]

**Comment by jrose-apple** [comment]
Uh, hm, and we don't put them all on one line anymore. Oops.

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test

File: test/ParseableInterface/synthesized.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  // CHECK-NEXT: case a, b, c
+  case a, b = 5, c
+  // CHECK-NEXT: public typealias RawValue = Swift.Int
+  // CHECK-NEXT: public var hashValue: Swift.Int {
+  // CHECK-NEXT:   get{{$}}
+  // CHECK-NEXT: }
+  // CHECK-NEXT: public func hash(into hasher: inout Swift.Hasher)
+  // CHECK-NEXT: @inlinable public init?(rawValue: Swift.Int)
+  // CHECK-NEXT: public var rawValue: Swift.Int {
+  // CHECK-NEXT:   @inlinable get{{$}}
```
Comment (jrose-apple) [inline test/ParseableInterface/synthesized.swift]: This is a little unfortunate, I guess. (It's a synthesized body, so we don't know how to print it.)

**Comment by brentdax** [comment]
@swift-ci please smoke test linux platform

**Comment by jrose-apple** [comment]
@swift-ci please smoke test linux platform

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test Linux

**Pull request merged**